### PR TITLE
Hide secrets command

### PR DESCRIPTION
**Pull request by fuskovic**

### CODE REVIEW

**Comment by cmoog** [comment]
Let's also print a warning message saying that it's deprecated.

File: ci/integration/secrets_test.go [hunk 1]
```diff
@@ -1,81 +1,80 @@
...
-	"cdr.dev/coder-cli/pkg/tcli"
+	//"cdr.dev/coder-cli/pkg/tcli"
 )
 
 func TestSecrets(t *testing.T) {
-	t.Parallel()
-	run(t, "secrets-cli-tests", func(t *testing.T, ctx context.Context, c *tcli.ContainerRunner) {
-		headlessLogin(ctx, t, c)
+	// t.Parallel()
+	// run(t, "secrets-cli-tests", func(t *testing.T, ctx context.Context, c *tcli.ContainerRunner) {
```
Comment (cmoog) [inline ci/integration/secrets_test.go:13]: let's call `t.Skip` rather than commenting them out

File: internal/cmd/secrets.go [hunk 2]
```diff
@@ -17,9 +17,19 @@ import (
...
+		Long:   "Interact with secrets objects owned by the active user.",
+		Hidden: true,
+		PersistentPreRun: func(cmd *cobra.Command, args []string) {
+			clog.LogWarn(
+				"The 'secrets' command is now deprecated",
+				"It will be removed in the next minor release",
+			)
+		},
+		Run: func(cmd *cobra.Command, args []string) {
+			_ = cmd.Help()
```
Comment (cmoog) [inline internal/cmd/secrets.go:31]: is this needed?

**Review by cmoog — APPROVED** [review]

File: internal/cmd/secrets.go (hunk 2 above)
Comment (fuskovic) [reply internal/cmd/secrets.go:31]: I noticed the `persistentPreRun` wouldn't run for `coder secrets` without a `Run`.

Once I added it then it worked but only printed the warning in the `persistentPreRun` so I added this particular line to make sure the help was there too.

**Pull request merged**